### ISSUE TITLE
Menu button is not labelled

### ISSUE THREAD

**Issue by yatil** [w3c/web-roadmaps#230]
```
<button id="side-nav-btn" type="button" name="sidenav-opener">
<img src="../assets/img/menu.svg" height="15" alt="">
</button>
```

* The button should have an alternative text (“open menu”)
* There should be a change of the alternative text when the menu is open, like “close menu”.
